### PR TITLE
Check liveblog updates is an array first

### PR DESCRIPTION
**Pull request by benbarnett**
Currently throwing an error on frontpage, probably due to an empty liveblog.

 🐿 v2.5.16

### CODE REVIEW

**Comment by leggsimon** [comment]
Worth adding `&& this.data.updates[0]`? Considering that's what is undefined

File: src/presenters/teaser-presenter.js [hunk 1]
```diff
@@ -366,7 +366,7 @@ const TeaserPresenter = class TeaserPresenter {
 	// returns publishedDate, status, classModifier
 	liveBlog () {
 		return {
-			publishedDate: this.data.updates && this.data.updates[0].date,
+			publishedDate: this.data.updates && Array.isArray(this.data.updates) && this.data.updates[0].date,
```
Comment (leggsimon) [inline src/presenters/teaser-presenter.js]: Worth adding `&& this.data.updates[0]`? Considering that's what is undefined. It could be an empty Array?

Comment (benbarnett) [reply src/presenters/teaser-presenter.js]: Yeah true, I *think* an empty array would just return undefined by that stage, but that is probably clearer...

Comment (leggsimon) [reply src/presenters/teaser-presenter.js]:
```

**Pull request merged**